### PR TITLE
Added adjustments for accelerometer trim

### PR DESCRIPTION
**Pull request by egonl**
Added two adjustments for trimming accelerometer pitch and roll.

image

### CODE REVIEW

File: src/main/fc/rc_adjustments.h [hunk 1]
```diff
@@ -131,6 +135,7 @@ typedef enum {
     ADJUSTMENT_TYPE_PROF  = BIT(1),
     ADJUSTMENT_TYPE_GOV   = BIT(2),
     ADJUSTMENT_TYPE_MIX   = BIT(3),
+    ADJUSTMENT_TYPE_ACC   = BIT(4),
```
Comment (rotorflight) [inline src/main/fc/rc_adjustments.h:138]: TYPE_NONE is for the parameters that don't need initialisation.
Don't add a new one for the purpose.

File: src/main/fc/rc_adjustments.c [hunk 2]
```diff
@@ -155,6 +157,9 @@ static const adjustmentConfig_t adjustmentConfigs[ADJUSTMENT_FUNCTION_COUNT] =
     ADJ_CONFIG(CROSS_COUPLING_GAIN,     PROF,  0, 250),
     ADJ_CONFIG(CROSS_COUPLING_RATIO,    PROF,  0, 200),
     ADJ_CONFIG(CROSS_COUPLING_CUTOFF,   PROF,  1, 250),
+
+    ADJ_CONFIG(ACC_TRIM_PITCH,          ACC,   -300, 300),
+    ADJ_CONFIG(ACC_TRIM_ROLL,           ACC,   -300, 300),
```
Comment (rotorflight) [inline src/main/fc/rc_adjustments.c:162]: Use NONE type.

**Review by rotorflight — CHANGES_REQUESTED** [review]

File: src/main/fc/rc_adjustments.h (hunk 1 above)
Comment (egonl) [reply src/main/fc/rc_adjustments.h]: Ok, good to know. It has been fixed.

File: src/main/fc/rc_adjustments.c (hunk 2 above)
Comment (egonl) [reply src/main/fc/rc_adjustments.c]: Done, thanks.

**Review by rotorflight — APPROVED** [review]

**Pull request merged**